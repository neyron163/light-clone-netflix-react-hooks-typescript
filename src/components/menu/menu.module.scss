@import '../variable.scss';

.menu {
    .active {
        font-weight: var(--font-weight-bold);
    }
}