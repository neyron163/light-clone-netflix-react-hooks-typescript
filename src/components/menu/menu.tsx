import React from "react";

import s from "./menu.module.scss";

export const Menu: React.FC = () => {
  return (
    <nav className={s.menu}>
      <ul className={s.list}>
        <li className={s.active}>
          <a href="/">Home</a>
        </li>
        <li>
          <a href="#">Tv Shows</a>
        </li>
        <li>
          <a href="#">Movies</a>
        </li>
        <li>
          <a href="#">Recently Added</a>
        </li>
        <li>
          <a href="#">My list</a>
        </li>
      </ul>
    </nav>
  );
};
