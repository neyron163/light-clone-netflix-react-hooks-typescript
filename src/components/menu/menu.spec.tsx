import React from "react";
import renderer from "react-test-renderer";

import { Menu } from "./menu";

describe("<Menu />", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Menu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
