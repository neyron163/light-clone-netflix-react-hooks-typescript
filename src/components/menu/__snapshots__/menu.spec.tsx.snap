// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> should render correctly 1`] = `
<nav
  className="menu"
>
  <ul
    className="list"
  >
    <li
      className="active"
    >
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        Tv Shows
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        Movies
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        Recently Added
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        My list
      </a>
    </li>
  </ul>
</nav>
`;
